src/mscorlib/shared/System/Char.cs

```diff
@@ -26,7 +26,7 @@ public struct Char : IComparable, IComparable<Char>, IEquatable<Char>, IConverti
         //
         // Member Variables
         //
-        private char _value;
+        private char m_value; // Do not rename (binary serialization)
```

Comment:
We were hitting a few issues and after the rename they were gone. Could be related to something else, needs more investigation. Due to the fact that we need to finish this for 2.0 we (@morganbr @danmosemsft @stephentoub @krwq) decided to keep the changes on the Primitives.